instagram/src/components/CommentSection.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import styled from "styled-components";
+
+
+// import './CommentSection.css';
+import dummyData from '../dummy-data';
+
+const CommentSectionDiv = styled.div `
+    line-height: 10px;
+    padding-left: 7px;
+`;
+
+class CommentSection extends Component {
+    constructor (props) {
+        super(props);
+        this.state = {
+            comments: this.props.commentData,
+            newComment:''
+        };
+    }
+
+
+    addNewComment = e => {
+        e.preventDefault();
+
+        const newComment = {
+            username: "Rory",
```

Comment:
```suggestion
            username: localStorage.getItem("username") || "Rory",
```
Sets the default to 'Rory' if there is no username in localStorage